jax/util.py

```diff
@@ -230,3 +230,7 @@ def wrap_name(name, transform_name):
 
 def extend_name_stack(stack, name=''):
   return stack + name + '/'
+
+
+class JaxDeprecationWarning(UserWarning):
+  pass
```

Comment:
is there a reason why we should use the builtin DeprecationWarning or FutureWarning?

Reply:
I avoided DeprecationWarning because it is hidden by default, so not very useful to users. FutureWarning would be fine, though.